src/main/webapp/app/modules/administration/user-management/user-management-update.tsx

```diff
@@ -110,7 +123,16 @@ export const UserManagementUpdate = (props: IUserManagementUpdateProps) => {
                 name={'authorities'}
                 options={authorityOptions}
                 defaultValue={[{ label: USER_AUTHORITY.ROLE_USER, value: USER_AUTHORITY.ROLE_USER }]}
+                onChange={(selectedOption, actionMeta) => {
+                  if (actionMeta.action === 'remove-value' && actionMeta.removedValue?.value === USER_AUTHORITY.ROLE_DEV) {
+                    setIsDev(false);
+                  }
+                  if (actionMeta.action === 'select-option' && selectedOption.some(option => option.value === USER_AUTHORITY.ROLE_DEV)) {
+                    setIsDev(true);
+                  }
+                }}
```

Comment:
I see that you are trying to show the Feature Flag dropdown only if the user is a dev. 
Note that the user management page is only accessible to admin (ROLE_ADMIN).
This means there is two scenarios:
1. An admin+dev user is editing another user (who has ROLE_DEV). If admin removes DEV role, it is still fine to show the feature flag dropdown because whether you show/not show the dropdown depends on if the current user is a dev or not.
2. An admin+dev user is editing their own account. They remove their own DEV role, but once you click the `Save` button, the page will refresh and the feature flag dropdown will not be visible anymore. In other words, the user is still a dev user until they save the DEV role removal changes.

tldr; I think it is fine to not have this onChange handler

Reply:
Looping in @jfkonecn to see what you think because I realized that this code was already approved.